src/Raven.Server/Documents/TombstoneCleaner.cs

```diff
@@ -199,8 +201,7 @@ internal TombstonesState GetState(bool addInfoForDebug = false)
 
                         foreach (var tombstone in subscriptionTombstones)
                         {
-                            if (addInfoForDebug)
-                                result.AddPerSubscriptionInfo(subscription.TombstoneCleanerIdentifier, tombstoneType, tombstone.Key, tombstone.Value);
+                            result.AddPerSubscriptionInfo(subscription.TombstoneCleanerIdentifier, tombstoneType, collection: tombstone.Key, etag: tombstone.Value);
```

Comment:
we are now adding that always, this looks suspicious

Reply:
My bad. Reverted this line...